cmd/storage-class.go

```diff
@@ -55,6 +55,16 @@ type storageClassConfig struct {
 	RRS      storageClass `json:"rrs"`
 }
 
+func (sCfg storageClassConfig) MarshalJSON() ([]byte, error) {
```

Comment:
What is this function, `MarshalJSON`,  for? 
Nobody is calling it in this PR's context. So, how  is it related to this PR?